samples/data-sealing/datasealing.edl

```diff
@@ -3,16 +3,16 @@
 
 enclave {
 
-    include "../shared.h"
+    include "common/shared.h"
     trusted {
-        public int seal_data(int sealPolicy, 
+        public int seal_data(int sealPolicy,
                              [in, size = opt_msg_len] unsigned char* opt_mgs,
                              size_t opt_msg_len,
                              [in, size = data_size] unsigned char* data,
                              size_t data_size,
                              [out] sealed_data_t** sealed_data,
```

Comment:
sealed_data_t should be annotated with size or should be user_check. @anakrish ?

Reply:
That might be true but I didn't touch these except maybe whitespace 😄

Reply:
OK, lets look at it another time?